tests/pbft_manager_test.cpp

```diff
@@ -179,67 +179,67 @@ void check_2tPlus1_validVotingPlayers_activePlayers_threshold(size_t committee_s
 struct PbftManagerTest : BaseTest {};
 
 // Test that after some amount of elapsed time will not continue soft voting for same value
-TEST_F(PbftManagerTest, terminate_soft_voting_pbft_block) {
-  auto node_cfgs = make_node_cfgs<20>(1);
-  auto nodes = launch_nodes(node_cfgs);
-
-  auto pbft_mgr = nodes[0]->getPbftManager();
-  pbft_mgr->stop();
-  std::cout << "PBFT manager stopped" << std::endl;
-
-  auto vote_mgr = nodes[0]->getVoteManager();
-
-  // Generate bogus votes
-  auto stale_block_hash = blk_hash_t("0000000100000000000000000000000000000000000000000000000000000000");
-  auto propose_vote = pbft_mgr->generateVote(stale_block_hash, propose_vote_type, 2, 1);
-  propose_vote->calculateWeight(1, 1, 1);
-  vote_mgr->addVerifiedVote(propose_vote);
-
-  pbft_mgr->setLastSoftVotedValue(stale_block_hash);
-
-  uint64_t time_till_stale_ms = 1000;
-  std::cout << "Set max wait for soft voted value to " << time_till_stale_ms << "ms..." << std::endl;
-  pbft_mgr->setMaxWaitForSoftVotedBlock_ms(time_till_stale_ms);
-  pbft_mgr->setMaxWaitForNextVotedBlock_ms(std::numeric_limits<uint64_t>::max());
-
-  auto sleep_time = time_till_stale_ms + 100;
-  std::cout << "Sleep " << sleep_time << "ms so that last soft voted value of " << stale_block_hash.abridged()
-            << " becomes stale..." << std::endl;
-  taraxa::thisThreadSleepForMilliSeconds(sleep_time);
-
-  std::cout << "Initialize PBFT manager at round 2 step 2" << std::endl;
-  pbft_mgr->setPbftRound(2);
-  pbft_mgr->setPbftStep(2);
-  pbft_mgr->resumeSingleState();
-  std::cout << "Into cert voted state in round 2..." << std::endl;
-  EXPECT_EQ(pbft_mgr->getPbftRound(), 2);
-  EXPECT_EQ(pbft_mgr->getPbftStep(), 3);
-
-  std::cout << "Check did not soft vote for stale soft voted value of " << stale_block_hash.abridged() << "..."
-            << std::endl;
-  bool skipped_soft_voting = true;
-  auto votes = vote_mgr->getVerifiedVotes();
-  for (const auto &v : votes) {
-    if (soft_vote_type == v->getType()) {
-      if (v->getBlockHash() == stale_block_hash) {
-        skipped_soft_voting = false;
-      }
-      std::cout << "Found soft voted value of " << v->getBlockHash().abridged() << " in round 2" << std::endl;
-    }
-  }
-  EXPECT_EQ(skipped_soft_voting, true);
-
-  auto start_round = pbft_mgr->getPbftRound();
-  pbft_mgr->resume();
-
-  std::cout << "Wait ensure node is still advancing in rounds... " << std::endl;
-  EXPECT_HAPPENS({60s, 50ms}, [&](auto &ctx) { WAIT_EXPECT_NE(ctx, start_round, pbft_mgr->getPbftRound()) });
-}
+// TEST_F(PbftManagerTest, terminate_soft_voting_pbft_block) {
```

Comment:
This is the only test that was using "only for tests" methods in pbft manager. So I commented test and this methods. I don't like idea that we have some public methods just for tests and maybe it will be easier to rework this test later. So I left it commented just to not forget about it